src/webgpu/util/f32_interval.ts

```diff
@@ -99,6 +110,33 @@ const kGreaterThanZeroInterval = new F32Interval(
   kValue.f32.positive.max
 );
 
+/** Representation of a vec2/3/4 of floating point intervals as an array of F32Intervals */
```

Comment:
I am quite pleased with how simple I was able to get this. It took a few tries to get it right.